collector/exchange.go

```diff
@@ -607,3 +631,12 @@ func (c *exchangeCollector) toLabelName(name string) string {
 func (c *exchangeCollector) msToSec(t float64) float64 {
 	return t / 1000
 }
+
+func find(slice []string, val string) bool {
+	for _, item := range slice {
+		if item == val {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Bit of a nit, but we've tried to keep generic utility functions in collector.go, since they are shared across the entire package.